src/includes/set-environment/unity.mdx

```diff
@@ -0,0 +1,15 @@
+```csharp
+using Sentry;
+
+// Add this to the SDK initialization callback
+options.Environment = "production";
+```
+
+```fsharp
+open Sentry
+
+// Add this to the SDK initialization callback
+options.Environment <- "production"
+```
+
+The SDK will set the environment by default depending on `Application.isEditor` to `editor` or `production` respectively.
```

Comment:
I think it's best if we don't add this file (meaning copy paste the whole thing from .NET) and simply add this extra line conditionally to the common content (as you did for the release and env pages).

If you still prefer adding this here, might as well remove F# then. And even write `SentryUnity(o => ` since we are specifically doing the unity part here.